### ISSUE TITLE
Add reason in the log for ban by ban button of report

### ISSUE THREAD

**Issue by Flowiee** [GreyWolfDev/Executrix#64]
When you ban someone by pressing ban button of report, add the reason of the report (if any) in the logged ban message

**Comment by Olgabrezel** [comment]
@Flowiee Good idea. And of course not only ban but all buttons (except for solve).
